script/build.sh

```diff
@@ -53,6 +53,7 @@ fi
 if [ "${assetSource}" = "local" ]; then
     echo "Building application assets"
     yarn build:webpack $webpackArgs
+    cp -v "${buildDir}generated/vendor.entry.js" "${buildDir}generated/shared-modules.entry.js"
```

Comment:
:thinking: This works for now, but we may need to circle back to this later. My tentative goal is to remove this script altogether when we move the content build out. If we still need this by then, we can come up with a better approach.

Reply:
Sounds good to me.  I'm _hoping_ we won't need this at that point.